content/learn-more/_index.html

```diff
@@ -0,0 +1,169 @@
+---
+title: RAPIDS.ai | Learn More
+description: Use cases and enterprise topics
+---
+
+  <!-- Hero Shortcode -->
+  {{< hero hero-text="Learn More" button-link="https://www.nvidia.com/en-us/data-center/products/ai-enterprise/" button-text="Learn about RAPIDS on NVIDIA AI Enterprise" >}}
+
+  <!-- Intro -->
+  <section class="section mb-6">
+    <div class="container">
+      <h1 class="has-text-centered mb-5"> Learn about the project, use cases, <br> and engagement with RAPIDS </h1>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-circle-info"></i> About </h2>
+          <p>RAPIDS is a collection of open source software libraries and APIs that gives you the ability to execute end-to-end
+            data science and analytics pipelines entirely on NVIDIA GPUs. <a href="#about">Jump To Section <i class="fa-solid fa-arrow-up-right"></i></a> </p>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-chart-user"></i> Use Cases </h2>
+          <p> Learn more about RAPIDS success stories, use cases, applications, research and other topics related to GPU accelerated data science. <a href="#use-cases">Jump To Section <i class="fa-solid fa-arrow-up-right"></i></a></p>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-message-exclamation"></i> Get Involved </h2>
+          <p> Get in contact with RAPIDS, enhanced support with NVIDIA enterprise services, and the broader developer community. <a href="#get-involved">Jump To Section <i class="fa-solid fa-arrow-up-right"></i></a></p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <div class="bottom-slant-container">
+    <div class="bottom-slant-down has-background-white"></div>
+  </div>
+
+  <!-- About  -->
+  <div id="about"></div>
+  <section class="section padding-top-slant padding-bottom-slant">
+    <div class="container">
+      <h1 class="mb-5"> About The Project</h1>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-house-laptop"></i> Familiar </h2>
+          <p>Utilizing NVIDIA CUDA primitives for low-level compute optimization, RAPIDS exposes GPU parallelism and high-bandwidth memory speed through user-friendly interfaces:</p>
+          <div class="content">
+            <ul> 
+              <li>Dataframe processing with Pandas like <a href="https://github.com/rapidsai/cudf" target="_blank">cuDF</a></li>
```

Comment:
I think the phrasing is off here. Shouldn't it be "Dataframe processing with cuDF like Pandas"? Same with all the ones below as well.

Reply:
changed to "comparable"